tools/importer/importer.py

```diff
@@ -148,10 +153,14 @@ def get_last_cherry_pick_sha(branch):
     lines = output.split('\n')
     for line in lines:
         if 'cherry picked from' in line:
-            sha = line.split(' ')[-1]
-            return sha[:-1]
+            sha = line.split(' ')[-1][:-1]
```

Comment:
why are you stripping the last character? is it a newline? If it is use `.strip()`instead.
```python
line = '11111 22222 33333 44444'
line.split(' ')[-1]
'44444'
line.split(' ')[-1][:-1]
'4444'
```

Reply:
git cherry-pick -x adds (cherry picked from commit <hash>​)
[:-1] trims the last bracket

Reply:
Ok.